test/account/utils/draft-ERC7579Utils.t.sol

```diff
@@ -298,8 +301,9 @@ contract ERC7579UtilsTest is Test {
 
         // GOOD
         this.callDecodeBatch(abi.encode(0));
-        // Note: Solidity also supports this even though it's odd
-        uint256[] memory _1 = abi.decode(abi.encode(0), (uint256[])); _1;
+        // Note: Solidity also supports this even though it's odd. Offset 0 means array is at the same location, which
+        // is interpreted as an array of length 0, which doesn't require any more data
+        abi.decode(abi.encode(0), (uint256[]));
```

Comment:
I'm getting a warning "Statement has no effect". This is why I had declared a variable.